modules/monitoring-enabling-alert-routing-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * monitoring/enabling-alert-routing-for-user-defined-projects.adoc
+
+[id="enabling-alert-routing-for-user-defined-projects_{context}"]
+= Enabling alert routing for user-defined projects
+
+You can enable alert routing for user-defined projects. By doing so, you enable users with the **alert-routing-edit** role to configure alert routing and receivers for user-defined projects in Alertmanager.
+
+.Prerequisites
+
+* You have enabled user-defined monitoring.
```

Comment:
You have enabled user-defined projects monitoring

Reply:
Fixed